Include/internal/pycore_runtime.h

```diff
@@ -160,6 +160,7 @@ typedef struct pyruntimestate {
     struct _types_runtime_state types;
 
     /* All the objects that are shared by the runtime's interpreters. */
+    struct _Py_cached_objects cached_objects;
```

Comment:
It would probably be nicer to debuggers to move this to the end of the struct, because I _think_ they like to poke at this struct (@pablogsal can confirm either way, I think). That said, other changes scheduled for 3.12.1 also change the layout of this struct, so maybe it no longer matters :S

Reply:
Yeah, although most of the relevant fields are already before this one and the next one is `struct _Py_static_objects static_objects;` which is already huge, so I don't think changing this would help a lot.

Reply:
I'm fine with it either way.  I'll plan on moving it to the end unless I hear otherwise.